File: Cpp/Day 8/inheritance/Assi1/emp.cpp
#include"emp.h"
    Emp::Emp()
    {
        eid=10;
        sal=10;
    }
    Emp::Emp(int id, const char *ch,int d,int m,int y,float sal):Person(ch,d,m,y)
    {
        this->eid=id;
        this->sal=sal;
    }
    void Emp::display()
    {
        cout<<endl<<"Employee id: "<<eid;
        Person::display();
        cout<<endl<<"Employee Salary: "<<sal;
    }

File: Java lab/polyoffice/src/polyoffice/staff/Emp.java
package polyoffice.staff;

import polyoffice.util.Person;

public class Emp extends Person {
	private int eid;
	private double salary;
	public Emp() {
		super();
	}
	public int getEid() {
		return eid;
	}
	public Emp(int eid, double salary) {
		super();
		this.eid = eid;
		this.salary = salary;
	}
	public Emp(String name,int dd, int mm, int yy,int eid, double salary) {
		super(name,dd,mm,yy);
		this.eid = eid;
		this.salary = salary;
	}
	public String toString() {
		return super.toString()+"\nEmpid : "+eid+"\nSalary : "+salary;
	}
	public void display()
	{	
		super.display();
		System.out.println("Employee id : "+eid+"\nSalary : "+salary);
	}
	public double calSalary()
	{
		return salary;
	}
}
